### DISCUSSION TITLE
Skywalking-Client-JS (Browser Monitoring)

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#11733] [General]
Hi Team,

Is it necessary that browser monitoring can only done on nodejs projects?
Kindly Help!!!

Regards,
Surya

**Comment by Fine0830** [comment]
It monitors web front-end projects, not node.js projects.

**Reply by SooryaMuthuraj** [reply]
I tried but unable to monitor, kindly suggest how to do it.

**Reply by Superskyyy** [reply]
Please at least elaborate what you mean by "unable"? along with a reproducible script.
